### DISCUSSION TITLE
Failed to read extensions descriptor

### DISCUSSION THREAD

**Discussion by luvs2pr0gram** [eclipse-tycho/tycho#1811] [Q&A]
I think this is an issue with my internal corporate repositories. Anything else to check? 

This one is very frustrating since I've made no changes to the config and was working fine a couple of months ago.

[WARNING] Failed to read extensions descriptor %USERHOME%\.mvn\extensions.xml: Plugin org.eclipse.tycho:tycho-build:2.7.4 or one of its dependencies could not be resolved: Failed to collect dependencies at org.eclipse.tycho:tycho-build:jar:2.7.4 -> org.eclipse.platform:org.eclipse.equinox.p2.director:jar:2.5.200 -> org.eclipse.platform:org.eclipse.equinox.p2.metadata:jar:2.3.100 -> org.eclipse.platform:org.eclipse.equinox.p2.core:jar:2.9.200 -> org.eclipse.platform:org.eclipse.core.runtime:jar:3.12.0 -> org.eclipse.platform:org.eclipse.equinox.preferences:jar:3.10.0 -> org.osgi.service:org.osgi.service.prefs:jar:[1.1.0,1.2.0)

**Comment by laeubi** [comment]
The first think you can try is using an empty local repository e.g with `-Dmaven.repo.local=/tmp/empty`

**Reply by luvs2pr0gram** [reply]
It seems like most of the dependencies download but this message comes up again and errors regarding non resolvable parent pom and missing child poms.

**Reply by laeubi** [reply]
Can you try 3.0.1 release when it is viable on central?

**Reply by luvs2pr0gram** [reply]
Same message.
